src/ring/middleware/charset.clj

```diff
@@ -0,0 +1,37 @@
+(ns ring.middleware.charset
+  "Middleware for automatically adding a charset to the content-type header in
+  response maps."
+  (:require [ring.util.response :as response]))
+
+(defn- text-based-content-type? [content-type-header]
+  "Is this a text-based content type? True for all text/ types and some
```

Comment:
No need for docstring on private vars if the function name makes the purpose self-evident.